app/models/components/course/groups_ability_component.rb

```diff
@@ -22,6 +22,7 @@ def allow_group_manager_manage_group
   end
 
   def course_group_manager_hash
-    { group_users: { user_id: user.id, role: Course::GroupUser.roles[:manager] } }
+    { group_users: { course_user_id: user.course_user_ids,
```

Comment:
[For my own use]

I tried some light debugging and ended up around this code block -> https://github.com/lowjoel/cancancan-squeel/blob/master/lib/cancancan/squeel/squeel_adapter.rb#L56-L59

```ruby
  def self.matches_relation?(subject, name, value)
    relation = subject.public_send(name)
    klass = subject.class.reflect_on_association(name).klass

    relation.where do
      expression, = CanCanCan::Squeel::ExpressionBuilder.build(self, klass, :==, value)
      expression
    end.any?
  end
```

Got lost at the line `relation.where do` (and some other information):
```ruby
subject = # group object
name = :group_users
value = { course_user: { user_id: 1}, role: 1 }
relation = # the group_user object
klass = Course::GroupUser
```